code/modules/power/cell.dm

```diff
@@ -61,6 +60,7 @@
 	return use(cell_amt) / CELLRATE
 
 /obj/item/weapon/cell/update_icon()
+	var/charge_status
```

Comment:
Is this part of the idea you were discussing last night for optimising cells inside apcs and chargers?

how does it work?